spring-cloud-dataflow-docs/src/main/asciidoc/configuration-database.adoc

```diff
@@ -0,0 +1,65 @@
+
+[[configuration-database-overview]]
+A relational database is used to store stream and task definitions as well as the state of executed tasks.
+Spring Cloud Data Flow provides schemas for *MariaDB*, *MySQL*, *Oracle*, *PostgreSQL*, *Db2*, *SQL Server*, and *H2*. The schema is automatically created when the server starts.
+
+NOTE: The JDBC drivers for *MariaDB*, *MySQL* (via the _MariaDB_ driver), *PostgreSQL*, *SQL Server* are available without additional configuration. To use any other database you need to put the corresponding JDBC driver jar on the classpath of the server as described <<#add-custom-driver,here>>.
+
+To configure a database the following properties must be set:
+
+* `spring.datasource.url`
+* `spring.datasource.username`
+* `spring.datasource.password`
+* `spring.datasource.driver-class-name`
+
+The `username` and `password` are the same regardless of the database. However, the `url` and `driver-class-name` vary per database as follows.
+
+[frame="none"]
+[cols="a,a,a,a"]
+[cols="10%,30%,20%,10%"]
```

Comment:
sexy table!

Reply:
You can get an idea of what it looks like w/ the attached PDF. Also , note the sexy footnote on the MySQL8 url info